src/features/search/components/OrganisationFull.tsx

```diff
@@ -32,44 +43,147 @@ export const OrganisationFull = ({
+
+              <WrapItem key={data.fieldOfStudy}>
+                <Tag tag={data.fieldOfStudy} />
+              </WrapItem>
+            </Wrap>
+          </Heading>
+          {data.logoUrl ? (
+            <Image
+              src={data.logoUrl}
+              width={100}
+              height={100}
+              style={{
+                margin: "0.6rem",
+                height: "6.25rem",
+                width: "6.25rem",
+                objectFit: "contain",
+              }}
+              alt={`Logo ${data.logoUrl || data.name}`}
+            />
+          ) : null}
+        </Flex>
+      </Box>
+
       <Wrap mt={2}>
         {data.Tags.map((tag) => (
           <WrapItem key={tag.text}>
             <Tag tag={tag.text} />
           </WrapItem>
         ))}
       </Wrap>
-      <Text mt={8}>{data.description}</Text>
       <br />
+      <HStack spacing={4}>
+        <Wrap>
+          {data.ContactMethods.map((contactMethod) => (
+            <Button
+              key={contactMethod.id}
+              leftIcon={getIconForContactType(contactMethod.contactType)}
+              colorScheme="gray"
+              variant="outline"
+              onClick={() =>
+                handleContactMethodClick(contactMethod.contactLink)
+              }
+            >
+              {getContactFixedName(contactMethod.contactType)}
+            </Button>
+          ))}
+        </Wrap>
+      </HStack>
+
+      <br />
+      <Heading as="h2" size="md" mt={4} mb={2}>
+        📰 Kim jesteśmy, co robimy 📰
+      </Heading>
       <Text>{data.longDescription}</Text>
       {data.ContactMethods.length > 0 ? (
         <>
           <Heading as="h2" size="md" mt={4} mb={2}>
-            Kontakt
+            🔥 Zdobywane umiejętności i wyzwania członków zespołu! 🔥
+          </Heading>
+          {data.skillsAndChallenges}
+          <Heading as="h2" size="md" mt={4} mb={2}>
+            ✨ Wyróżniamy się tym, że... ✨
           </Heading>
-          <List>
-            {data.ContactMethods.map((contactMethod) => (
-              <ListItem key={contactMethod.id}>
-                <HStack>
-                  <Text textTransform="capitalize">
-                    {contactMethod.contactType}:
-                  </Text>
-                  <Link
-                    href={contactMethod.contactLink}
-                    target="_blank"
-                    color="gray"
-                  >
-                    {contactMethod.contactLink}
-                  </Link>
-                </HStack>
-              </ListItem>
+          {data.distinguishingFeatures}
+          <Heading as="h2" size="md" mt={4} mb={2}>
+            🏆 Największe sukcesy uczelnianej organizacji studenckiej! 🏆
+          </Heading>
+          {data.achievements}
+          <Heading as="h2" size="md" mt={4} mb={2}>
+            🌟 Obszary zainteresowań studentów dołączających do naszej
+            organizacji! 🌟
+          </Heading>
+          {data.areasOfInterest}
```

Comment:
bym wrzucił te opisy w <Text></Text>

Reply:
Te opisy już są w headingach, czy to nie wystarczy?

Reply:
Bo mi też zależy, aby te opisy były trochę większe niż tekst pod nimi, aby się wyróżniały

Reply:
nie no, te headingi są jak najbardziej spoko, chodzi mi tylko, żeby ten tekst pod nimi wrzucić w <Text></Text> czyli w tym przypadku
```tsx
import {Text} from "@chakra-ui/react"
...
<Text>{data.areasOfInterest}</Text>
...
```